lib/util.js

```diff
@@ -150,11 +150,13 @@ exports.isNumber = function (value)
  *
  * @returns {Boolean}
  */
-exports.isPrivateKey = function (value)
-{
+exports.isPrivateKey = function (value) {
+  value = value.trim();
   // The private key is expected to be decrypted when set in the connection string
-  return (value.startsWith("-----BEGIN PRIVATE KEY-----") &&
-    value.endsWith("\n-----END PRIVATE KEY-----\n"));
+  // secret scanner complains about first check since it looks like private key, but it's only check
+  // pragma: allowlist nextline secret
+  return (value.startsWith('-----BEGIN PRIVATE KEY-----') &&
+    value.endsWith('\n-----END PRIVATE KEY-----'));
```

Comment:
Maybe instead of overwriting function param (this is not the best practice I suppose), maybe better do it here `value.trim().endsWith(...)`?

Reply:
unfortnuatelly it is often pattern in javascript - but yes, we can omit it